include/berry_pots_app_internal.h

```diff
@@ -0,0 +1,114 @@
+#ifndef POKEHEARTGOLD_BERRY_POTS_APP_INTERNAL_H
+#define POKEHEARTGOLD_BERRY_POTS_APP_INTERNAL_H
+#include "unk_02023694.h"
+#include "game_stats.h"
+#include "overlay_16.h"
+#include "unk_0203E348.h"
+
+enum BerryPotsApp_022036FC_Action {
+    BERRY_POTS_APP_022036FC_ACTION_0,
+    BERRY_POTS_APP_022036FC_ACTION_1,
+    BERRY_POTS_APP_022036FC_ACTION_2,
+    BERRY_POTS_APP_022036FC_ACTION_3,
+    BERRY_POTS_APP_022036FC_ACTION_4,
+    BERRY_POTS_APP_022036FC_ACTION_5,
+    BERRY_POTS_APP_022036FC_ACTION_6,
+    BERRY_POTS_APP_022036FC_ACTION_7,
+    BERRY_POTS_APP_022036FC_ACTION_8,
+    BERRY_POTS_APP_022036FC_ACTION_9,
+    BERRY_POTS_APP_022036FC_ACTION_10,
+    BERRY_POTS_APP_022036FC_ACTION_11,
+    BERRY_POTS_APP_022036FC_ACTION_12,
+    BERRY_POTS_APP_022036FC_ACTION_13,
+    BERRY_POTS_APP_022036FC_ACTION_14,
+    BERRY_POTS_APP_022036FC_ACTION_15,
+    BERRY_POTS_APP_022036FC_ACTION_16,
+};
+
+typedef struct BerryPotsAppData_UnkSub20 {
+    u8 berryId;
+    u8 growthStage;
+    u8 unk2;
+    u8 moisture;
+    u8 mulch;
+    u8 quantityOrYieldMaybe;
+    u16 unk6;
+    u16 unk8;
+    u8 soilState;
+    u8 unkB;
+    u8 unkC;
+    u8 unkD;
+    Sprite *soilSpriteMaybe;
+} BerryPotsAppData_UnkSub20;
+
+typedef struct BerryPotsAppData {
+    HeapID heapId;
+    BerryPotsAppArgs *args;
+    void *unk8;
+    Bag *bag;
+    GAME_STATS *stats;
+    BerryPot *berryPots;
+    struct GF_RTC_DateTime *berryDatetime;
+    UnkStruct_ov16_022014A0 *berryGrowthProperties;
+    BerryPotsAppData_UnkSub20 unk20[MAX_BERRY_POT];
+    u16 state70;
+    u16 state72;
+    u32 unk74;
+    u8 textFrameDelay;
+    u8 frame;
+    u8 unk7A;
+    u8 unk7B;
+    u8 unk7C;
+    u8 runningTasks;
+    u8 unk7E;
+    u8 waitFrames;
+    u8 unk80;
+    u8 unk81;
+    u16 totalBerryQuantity;
+    u16 totalMulchQuantity;
+    BgConfig *bgConfig;
+    void *unk8C;
+    u8 unk90[0x9C - 0x90];
```

Comment:
perform subtraction

Reply:
this was copy-paste from tgsm so let's yell at him and also whoever approved his work

Reply:
> and also whoever approved his work

GUESS WHO, IT'S YOU!

Reply:
I miss things omg